templates/main.php

```diff
@@ -96,7 +96,7 @@
 	 */
 	do_action( 'bp_before_group_home_content' ); ?>
 
-	<div id="item-header" role="complementary">
+	<div id="item-header" role="complementary" data-bp-item-id="<?php bp_group_id(); ?>" data-bp-item-component="groups" class="groups-header single-headers">
```

Comment:
Could we use `esc_attr` for `bp_group_id()`?

Reply:
Its internally escaped, that's why I haven't echoed it here

Reply:
Ok